packages/avro-ts/src/index.ts

```diff
@@ -251,28 +255,45 @@ const fullyQualifiedName = (context: Context, type: schema.RecordType) => {
   return currentNamespace ? `${currentNamespace}.${type.name}` : type.name;
 };
 
-export const printAstNode = (node: Result): string => {
+export const printAstNode = (importLines: Array<string>, node: Result): string => {
```

Comment:
This function is exported, so the change in args isn't backwards compatible - ie. it'd be a major semver bump.

To keep it a minor-version bump, we could:

* Re-arrange the args to keep `node: Result` as the first arg
* Or maybe make the second arg an object with a number of fields containing extra info such as the import lines, allowing more fields to be added in the future

...though the function doesn't look to be used outside this file (at least [in OVO](https://github.com/search?q=org%3Aovotech+printAstNode&type=Code)). As such, we could possibly remove the `export`, make a major version bump and call the presence of the `export` a bug.

Reply:
@ivank Can you recall the original reason for exporting this function?

Reply:
That's a very good point. I could also keep the exported version as a single argument function, make it console.error a deprecation notice when called, and only bump a minor version.

`avroTs` would use a version of that function that is not exported, does not contain the deprecation warning and support more arguments.

Reply:
Melikey the deprecation notice idea.

Reply:
Changes applied.

Reply:
that function was exported as I was thinking of testing the function itself, but opted for more functional tests later